charts/incubator/ghostfolio/questions.yaml

```diff
@@ -0,0 +1,100 @@
+# Include{groups}
+portals:
+  open:
+# Include{portalLink}
+questions:
+# Include{global}
+# Include{workload}
+# Include{workloadDeployment}
+# Include{replicas1}
+# Include{podSpec}
+# Include{containerMain}
+# Include{containerBasic}
+# Include{containerAdvanced}
+# Include{containerConfig}
+# Include{podOptions}
+  - variable: alpha_vantage_api_key
+    group: "App Configuration"
+    label: "alpha_vantage_api_key"
+    description: "https://www.alphavantage.co/ API key"
+    schema:
+      type: string
+      default: ""
+# Include{serviceRoot}
+  - variable: main
+    label: Main Service
+    description: The Primary service on which the healthcheck runs, often the webUI
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+# Include{serviceSelectorLoadBalancer}
+# Include{serviceSelectorExtras}
+  - variable: main
+    label: Main Service Port Configuration
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: port
+          label: Port
+          description: This port exposes the container port on the service
+          schema:
+            type: int
+            default: 13021
+            required: true
+# Include{serviceExpertRoot}
+# Include{serviceExpert}
+# Include{serviceList}
+# Include{persistenceRoot}
+  - variable: config
+    label: App Config Storage
+    description: Stores the Application Config.
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+# Include{persistenceBasic}
+# Include{persistenceList}
+# Include{ingressRoot}
+  - variable: main
+    label: Main Ingress
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+# Include{ingressDefault}
+# Include{ingressTLS}
+# Include{ingressTraefik}
+# Include{ingressList}
+# Include{securityContextRoot}
+  - variable: runAsUser
+    label: runAsUser
+    description: The UserID of the user running the application
+    schema:
+      type: int
+      default: 0
+  - variable: runAsGroup
+    label: runAsGroup
+    description: The groupID of the user running the application
+    schema:
+      type: int
+      default: 0
+# Include{securityContextContainer}
+# Include{securityContextAdvanced}
+# Include{securityContextPod}
+  - variable: fsGroup
+    label: fsGroup
+    description: The group that should own ALL storage.
+    schema:
+      type: int
+      default: 568
+# Include{resources}
+# Include{metrics}
+# Include{prometheusRule}
+# Include{advanced}
+# Include{addons}
+# Include{codeserver}
+# Include{netshoot}
+# Include{vpn}
+# Include{documentation}
```

Comment:
Look at romm, jackett and other stable/enterprise apps for what the questions.yaml can look like.

just use the default env template and not a custom section.

Reply:
@xstar97 thank you! now im getting a different error. might have to do with the pipeline ?

Reply:
ah no its fixed. i've taken a look at romm or jacket and upated the questions. however looks like its failing to install. i wonder if i can test this locally :(